src/hotspot/os/bsd/attachListener_bsd.cpp

```diff
@@ -248,7 +248,7 @@ int BsdAttachListener::init() {
 //
 BsdAttachOperation* BsdAttachListener::read_request(int s) {
   char ver_str[8];
-  sprintf(ver_str, "%d", ATTACH_PROTOCOL_VER);
+  os::snprintf(ver_str, sizeof(ver_str), "%d", ATTACH_PROTOCOL_VER);
```

Comment:
We later use `strlen(ver_str)` where we could instead use the result of `os::snprintf`.